staging/src/k8s.io/apiserver/pkg/cel/limits.go

```diff
@@ -48,5 +48,7 @@ const (
 	// MinNumberSize is the length of literal 0
 	MinNumberSize = 1
 
+	// MaxFormatSize is the maximum size we allow for format strings
+	MaxFormatSize          = 64
```

Comment:
is this an existing limit or a new limit in this PR?

Reply:
It's a new limit but note that we only support a fixed set of formats (we don't allow arbitrary string inputs to be represented as format strings) and the set of formats we have is well below this limit, so this is non-breaking.